### PR TITLE
Fixed link to building a client in Readme.md

### PR DESCRIPTION
**Pull request by nelreina**

### CODE REVIEW

**Comment by goloroden** [comment]
Hey @nelreina 😊 

Thanks a lot for your pull request!

Two things:

- Could you please sign the CLA, so that I can merge it?
- And, if you want to, could you please add yourself to the list of collaborators in the `package.json` file?

**Comment by nelreina** [comment]
Hi @goloroden 
Done and Done!

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ $ npx roboter
 
 To run the tests on [BrowserStack](https://www.browserstack.com/), you need to define the environment variable `SELENIUM_ENV` and set it to the value `browserstack`.
 
-*Please note that you need to have the environment variables `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` set to your account's credentials.*
+_Please note that you need to have the environment variables `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` set to your account's credentials._
```
[the remark this answers is not in the document]
Comment (nelreina) [reply README.md]: I am using Vscode as my text editor and when I saved the file vscode did change the * into _ .  [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) They are equivalent. I don't know why vscode changes them.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ wolkenkit-client is a client to access wolkenkit applications.
 
 ## Documentation
 
-See [https://docs.wolkenkit.io/latest/reference/building-a-client/connecting-to-an-application/](Buildling a client) on how to use this module to build a client that connects to a [wolkenkit](https://www.wolkenkit.io) application.
+See [Buildling a client](https://docs.wolkenkit.io/latest/reference/building-a-client/connecting-to-an-application) on how to use this module to build a client that connects to a [wolkenkit](https://www.wolkenkit.io) application.
```
[the remark this answers is not in the document]
Comment (nelreina) [reply README.md]: I fixed the typo

**Pull request merged**